<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCustomerWidget</class>
 <widget class="QDialog" name="AddCustomerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>470</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>470</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>添加会员</string>
  </property>
  <property name="windowIcon">
   <iconset theme=":/MainWindow/Resources/addCustomer.png"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="2">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>239</width>
       <height>509</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>28</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>华文行楷</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>添加会员</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>华文行楷</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>姓名</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_25">
           <property name="font">
            <font>
             <family>华文行楷</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>手机</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_22">
           <property name="font">
            <font>
             <family>华文行楷</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>微信</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_19">
           <property name="font">
            <font>
             <family>Algerian</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>QQ</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_21">
           <property name="font">
            <font>
             <family>华文行楷</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>地址</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_23">
           <property name="font">
            <font>
             <family>华文行楷</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>性别</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_24">
           <property name="font">
            <font>
             <family>华文行楷</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>会员号</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_26">
           <property name="font">
            <font>
             <family>华文行楷</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>积分</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLineEdit" name="name_lineEdit"/>
         </item>
         <item>
          <widget class="QLineEdit" name="tel_lineEdit"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="weChat_lineEdit"/>
           </item>
           <item>
            <widget class="QCheckBox" name="sameAsTel_checkBox">
             <property name="text">
              <string>同手机</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLineEdit" name="qq_lineEdit"/>
         </item>
         <item>
          <widget class="QLineEdit" name="addr_lineEdit"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QRadioButton" name="female_radioButton">
             <property name="text">
              <string>女</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="male_radioButton">
             <property name="text">
              <string>男</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLineEdit" name="vip_lineEdit">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="score_lineEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="font">
        <font>
         <family>华文行楷</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="title">
        <string>度数</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>近视</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>散光</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>左</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="left_myopia_comboBox">
          <item>
           <property name="text">
            <string>0</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>25</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>50</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>75</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>100</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>125</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>150</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>175</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>225</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>250</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>275</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>300</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>325</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>350</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>375</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>400</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>425</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>450</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>475</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>500</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>550</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>600</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>650</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>700</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>750</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>800</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>850</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>900</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>950</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1000</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QComboBox" name="left_astigmatism_comboBox">
          <item>
           <property name="text">
            <string>0</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>25</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>50</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>75</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>100</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>125</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>150</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>175</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>225</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>250</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>275</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>300</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>325</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>350</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>375</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>400</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>425</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>450</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>475</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>500</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>550</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>600</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>650</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>700</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>750</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>800</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>850</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>900</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>950</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1000</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>右</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="right_myopia_comboBox">
          <item>
           <property name="text">
            <string>0</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>25</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>50</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>75</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>100</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>125</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>150</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>175</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>225</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>250</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>275</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>300</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>325</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>350</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>375</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>400</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>425</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>450</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>475</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>500</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>550</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>600</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>650</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>700</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>750</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>800</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>850</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>900</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>950</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1000</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QComboBox" name="right_astigmatism_comboBox">
          <item>
           <property name="text">
            <string>0</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>25</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>50</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>75</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>100</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>125</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>150</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>175</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>225</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>250</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>275</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>300</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>325</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>350</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>375</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>400</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>425</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>450</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>475</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>500</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>550</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>600</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>650</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>700</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>750</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>800</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>850</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>900</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>950</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1000</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="add_pushButton">
         <property name="font">
          <font>
           <family>华文行楷</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>添加</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cancle_pushButton">
         <property name="font">
          <font>
           <family>华文行楷</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>335</width>
       <height>47</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>239</width>
       <height>509</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>335</width>
       <height>47</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
